allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -506,30 +505,56 @@ def _build_units_table(self, units_table):
             & (table['quality'] == 'good')
         ]
 
-        table = table.drop(columns=['local_index_unit', 'quality', 'valid_data'])
+        # table = table.drop(columns=['local_index_unit', 'quality', 'valid_data'])
         return table.sort_values(by=['probe_description', 'probe_vertical_position', 'probe_horizontal_position'])
 
 
+    def _build_nwb1_waveforms(self, mean_waveforms):
```

Comment:
This functionality needs to move to the API layer. This can be another issue, though.